conans/client/conf/config_installer.py

```diff
@@ -258,10 +262,25 @@ def configuration_install(app, uri, verify_ssl, config_type=None,
         _save_configs(configs_file, configs)
 
 
+def _is_scheduled_intervals(file, interval):
```

Comment:
I moved to here for better simplification, and be able to mock in functional tests.